part-spec/power.json

```diff
@@ -697,6 +697,516 @@
           }
         }
       }
+    },
+    {
+      "title": "display backlight driver",
+      "type": "object",
+      "required": [
+        "componentID",
+        "ioutMax",
+        "vinMin",
+        "vinMax"
+      ],
+      "properties": {
+        "componentID": {
+          "description": "common component identifying information, such as mpn",
+          "$ref": "definitions.json#/componentID"
+        },
+        "vinTyp": {
```

Comment:
would it make sense to combine Typ, Min, Max into an object for reuse since we see this pattern being used often?

Reply:
No strong preference from my side. Whatever works better for tools developers.  My proposal is to release this as is for the rev 0.8 but to come back in rev 0.9 and make the change for all  the components we've defined so far.

Reply:
Yeah I think it makes sense to push this to v0.9.